packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -91,6 +91,10 @@ export const VSelect = genericComponent<new <
   name: 'VSelect',
 
   props: {
+    valueComparator: {
+      type: Function as PropType<typeof deepEqual>,
+      default: deepEqual,
+    },
```

Comment:
This can go in makeSelectProps can't it?

Reply:
Yes! Good point, I added it.